test/unit/spec/media/track/videotrack.js

```diff
@@ -124,9 +124,17 @@ describe('VideoTrack', () => {
           sinon.assert.calledOnce(videoTrack._updateElementsMediaStreamTrack);
         });
 
-        it('should start capturing frames', () => {
+        it('should start capturing frames after setting the processor', () => {
           sinon.assert.calledOnce(videoTrack._captureFrames);
         });
+
+        it('should start capturing frames whenever mediaStreamTrack unmutes', () => {
```

Comment:
What about tests for `enabled`?

Reply:
Since `.enable` already calls captureFrames, we already have that test [here](https://github.com/twilio/twilio-video.js/pull/1374/files#diff-7925908b69cda71b02214829e73cb7b620e9515975759a6e44e96fd3c8c2dea3R90).